### DISCUSSION TITLE
textOverflow配置失效，文字向左侧溢出

### DISCUSSION THREAD

**Discussion by wzdong26** [antvis/S2#2940] [General]
option中配置：
![image](https://github.com/user-attachments/assets/44e542d1-0382-4b60-a9ba-0747a848bbcc)
异常表现：
![20241024163858_rec_](https://github.com/user-attachments/assets/feae7593-aa23-41a0-9c04-c8d92e500089)

**Comment by wzdong26** [comment]
设置了layoutWidthType: LayoutWidthType.Compact就会复现

**Comment by lijinke666** [comment]
已知问题: https://github.com/antvis/S2/issues/2900, 待排期修复

**Reply by lijinke666** [reply]
升级下最新版本再试试, 没有复现

![Kapture 2024-10-29 at 17 49 48](https://github.com/user-attachments/assets/f04a5bbd-4b63-466b-bd4e-cf4089781de8)


如果还有问题, 请按照模版提 issue, 提供版本和可复现链接.
